src/scss/mixins/_common.scss

```diff
@@ -5,19 +5,18 @@
 
 /// Common field styles
 @mixin oFormsCommonFieldBase {
-	@include oTypographyMargin($top: 3);
+	@include oColorsFor(o-forms-field-standard);
+	margin-top: $_o-forms-spacing;
```

Comment:
🤔 as discussed...

Seemingly arbitrary padding like this can be swapped out for a size intune with our vertical rhythm. In this case the height of inputs are set to 40px anyway:
https://github.com/Financial-Times/o-forms/pull/170/files#diff-73b5b8a1f62da0fc9ef102b351d0fee9R34